app/controllers/listings_controller.rb

```diff
@@ -559,4 +566,10 @@ def delivery_config(require_shipping_address, pickup_enabled, shipping_price, cu
       []
     end
   end
+
+  def create_listing_params(params)
```

Comment:
This could and should be implemented as pure function instead of modifying input params.